test/bats.bats

```diff
@@ -49,7 +49,16 @@ fixtures bats
 @test "summary passing and skipping tests" {
   run filter_control_sequences bats -p $FIXTURE_ROOT/passing_and_skipping.bats
   [ $status -eq 0 ]
-  [ "${lines[2]}" = "2 tests, 0 failures, 1 skipped" ]
+  [ "${lines[3]}" = "3 tests, 0 failures, 2 skipped" ]
+}
+
+@test "tap passing and skipping tests" {
+  run filter_control_sequences bats --tap $FIXTURE_ROOT/passing_and_skipping.bats
```

Comment:
Add double quotes around the `$FIXTURE_ROOT/passing_and-skipping.bats` path here and on line 77. Otherwise a `$FIXTURE_ROOT` with spaces in it will break the test. See ebb192e036284b5006ecac08343c72be6e93e19b from #8.

Reply:
@agent-0028 You good to make this change?

Reply:
Indeed. I'll take care of it.